dropwizard-migrations/src/main/java/io/dropwizard/migrations/CloseableLiquibase.java

```diff
@@ -29,6 +33,12 @@ public CloseableLiquibase(ManagedDataSource dataSource, String file) throws Liqu
 
     @Override
     public void close() throws Exception {
+        try {
+            database.close();
+        } catch (LiquibaseException e){
```

Comment:
I think it would be better to remove the `catch` block just let the exception propagate (the method already declares to throw `Exception`) and put the `dataSource.stop()` call into the `finally` block of the `try-finally` statement.